packages/mobile/src/harmony-native/components/CoverPhoto/CoverPhoto.tsx

```diff
@@ -0,0 +1,72 @@
+import type { ReactNode } from 'react'
+
+import type { CornerRadiusOptions } from '@audius/harmony'
+import { css } from '@emotion/native'
+import { BlurView } from '@react-native-community/blur'
+import type {
+  ImageBackgroundProps,
+  ImageSourcePropType,
+  ImageStyle,
+  StyleProp
+} from 'react-native'
+import { ImageBackground, View } from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+
+import { useTheme } from '../../foundations/theme'
+
+export type CoverPhotoProps = {
+  profilePicture?: ImageSourcePropType
+  coverPhoto?: ImageSourcePropType
+  style?: StyleProp<ImageStyle>
+  children?: ReactNode
+  topCornerRadius?: CornerRadiusOptions
+} & Omit<ImageBackgroundProps, 'source'>
+
+export const CoverPhoto = (props: CoverPhotoProps) => {
```

Comment:
Are you thinking we add this to harmony web at some point as well?

Reply:
yeah i wonder what you think about this? its kinda like our "molecule" components. i can image that a 3rd party may want to use our cover-photo cause it comes with the built-in logic to do the blur etc. what do you think?